src/new-components/alert/alert.tsx

```diff
@@ -28,19 +29,34 @@ function Alert({ id, children, tone, closeLabel, onClose }: AlertProps) {
             id={id}
             role="alert"
             aria-live="polite"
-            padding="small"
+            borderRadius="standard"
             className={[styles.container, getClassNames(styles, 'tone', tone)]}
         >
             <Columns space="small" alignY="top">
                 <Column width="content">
                     <AlertIcon tone={tone} />
                 </Column>
-                <Column>{children}</Column>
-                {onClose != null ? (
+                <Column>
+                    <Box
+                        paddingY="xsmall"
+                        paddingRight={onClose != null && closeLabel != null ? undefined : 'small'}
+                    >
+                        {children}
+                    </Box>
+                </Column>
+                {onClose != null && closeLabel != null ? (
                     <Column width="content">
-                        <button type="button" onClick={onClose} aria-label={closeLabel}>
-                            <CloseIcon />
-                        </button>
+                        <Button
+                            variant="quaternary"
+                            size="small"
+                            onClick={onClose}
+                            aria-label={closeLabel}
+                            icon={<CloseIcon />}
+                            style={{
+                                // @ts-expect-error not sure how to make TypeScript understand custom CSS properties
+                                '--reactist-btn-hover-fill': 'transparent',
```

Comment:
This is outside of the scope of this ticket, but when our custom properties are more-or-less finalized we could do something like this: https://stackoverflow.com/a/65959390